man1/aur-build.1

```diff
@@ -373,6 +373,13 @@ See
 .BR gpg (1)
 for details.
 .
+.SS Rebuilding packages against updated dependencies
+It is sometimes needed to rebuild packages when their dependencies are
+updated, for example in case of dynamic library linking.
+.PP
+You can use https://github.com/maximbaz/rebuild-detector
```

Comment:
Groff has some special syntax for URIs too:
```
.UR https://\:github.com/\:maximbaz/\:rebuild-detector
.UE .
```

Reply:
Nice - seems like even named URLs work, I saw an example in `aur-pkglist`, updated 👍